slashing-protection/src/main/resources/migrations/postgresql/V6__add_db_version.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE database_version (
```

Comment:
Could just add this into the metadata table? Maybe that table should of been generic string, string to store all this slashing protection metadata.

Reply:
makes life really hard due to the non-null constraint on GVR in the metadata table :/